app/client/src/sagas/WidgetOperationSagas.tsx

```diff
@@ -528,14 +528,16 @@ export function* deleteAllSelectedWidgetsSaga(
       setTimeout(() => {
         if (bulkDeleteKey) {
           flushDeletedWidgets(bulkDeleteKey);
-          AppsmithConsole.info({
-            logType: LOG_TYPE.ENTITY_DELETED,
-            text: `${selectedWidgets.length} were deleted`,
-            source: {
-              name: "Group Delete",
-              type: ENTITY_TYPE.WIDGET,
-              id: bulkDeleteKey,
-            },
+          falttendedWidgets.map((e: any) => {
```

Comment:
nitpick: using entity names instead of shorthand
```suggestion
          falttendedWidgets.map((widget: any) => {
```